### ISSUE TITLE
[A11y] Playlist index page 

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/avalon#6351]
### Description

From the accessibility checks performed on playlist related pages in https://github.com/avalonmediasystem/avalon/issues/6312, the following were identified for playlist index page.
| Success criterion | WCAG 2.1 Level | Accessibility checker | Error | Relevant HTML element |
| --- | --- | --- | --- | --- |
| 4.1.2 Name, Role, Value | Level A | SiteImprove & WAVE| Form field missing a label | Tag filter drop-down menu |
| 1.4.3 Contrast (Minimum) | Level AA | SiteImprove & WAVE | Color contrast (2.85:1) doesn't meet min requirement | Disabled `Previous` button on the table pagination |
| 1.4.3 Contrast (Minimum) | Level AA | SiteImprove & WAVE | Color contrast (2.54:1) doesn't min requirement | Highlighted page number button on the table pagination |

### Done Looks Like
 - [ ] For the first issue: Since there is a label for this form element, we can probably use it to fix these issues by making the form label part of the drop-down making it accessible (Details of implementations: https://dequeuniversity.com/rules/axe/4.10/label?application=AxeFirefox)
 - [ ] Fix color contrast for table pagination elements

**Comment by joncameron** [comment]
These are resolved and no longer flagged in Siteimprove.
